Dizzy/Web Service/FirebaseWebService.swift

```diff
@@ -50,14 +56,14 @@ final class FirebaseWebService: WebServiceType {
     private func sendPostRequest<Response, Body>(resource: Resource<Response, Body>,
                                                  completion: @escaping (Result<Response>) -> Void) {
         if let json = resource.makeJson() {
-            databaseReference.child(resource.path).setValue(json) { (error, _) in
+            databaseReference.child("\(resource.path)/\(UUID())").setValue(json) { (error, _) in
```

Comment:
i am not sure that `UUID()` should be here, i think that every object that we want to save need to have id in it, what do you think about it?

Reply:
I'm not sure too, for now, I think this code is ok, I think that in future we will be more clever